lib/cfg-parser.c

```diff
@@ -378,3 +378,43 @@ cfg_process_yesno(const gchar *yesno)
     return TRUE;
   return FALSE;
 }
+
+gboolean
+cfg_process_list_of_literals(const gchar *input, GList **result)
+{
+  if (strchr(input, '"') || strchr(input, '\''))
+    {
+      fprintf(stderr, "Error: list of literals must not contain single or double quotes: %s\n", input);
+      return FALSE;
+    }
+
+  *result = NULL;
+
+  gchar *input_copy = g_strdup(input);
+  const gchar delimiters[] = ", \t\n";
+  const gchar *elem = strtok(input_copy, delimiters);
+  while (elem)
+    {
+      if (strlen(elem) > 0)
+        *result = g_list_append(*result, __normalize_key(elem));
+      elem = strtok(NULL, delimiters);
+    }
+
+  g_free(input_copy);
```

Comment:
Wouldn't `g_strsplit()` help here (both the usage and its return value)?

Reply:
The problem is we need to normalize the substring. If we used g_strsplit, we would have to reallocate every element.

Reply:
I overlooked the normalization part.